solutions/bio1/IncreasingPasswords/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { SCodeBlock, SText, STitle } from "components";
+
+export const IncreasingPasswords = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        From the constraint on $n$, we can see that it is not feasible to generate each possible password.
+        Instead, we should use dynamic programming to efficiently build the $n$-th password.
+      </SText>
+
+      <STitle>Solution</STitle>
+      <SText>
+        Let's first define a recursive function that gets the number of passwords of length $n$, with all of its letters being greater than $c$.
+        To do this, we can first define the base case for when the length is 1. 
+        The number of ways to generate a password of length 1 with all characters greater than $c$: $36 - c$.
+        Otherwise, we can recursively call the function with the length $n-1$ and the character $c+i$ for all values of $i$ such that $c+i \leq 36$ (1-indexed).
```

Comment:
a bit pedantic but since there are other errors you might as well fix this too

write 'positive integer values of `$i$`'